fuzon-http/README.md

```diff
@@ -0,0 +1,23 @@
+# fuzon-http
+
+This is a web-server to deploy fuzon as a web-service.
+All ontologies are loaded once on server startup, and the indices are kept in memory.
+
```

Comment:
**suggestion**: around the top of the README I would add the gif you shared with us, maybe with a focus on the tool showing the top X matching terms. That's an easy way to see the tool in action and understand what it does. Alternatively the gif can also go to the main README to showcase it even more.

Reply:
Good idea! I've added a minimal (15 lines of bash) client example to the readme in f5b8595 to show how to interact with the server, and a gif of what it may look like.